README.md

```diff
@@ -433,6 +433,26 @@ build folder to your path:
 set PATH=C:\STL\out\x64\out\bin\amd64;%PATH%
 ```
 
+## Running tests with Address Sanitizer (ASan)
+
+You don't need any extra steps to run with test code and the code in STL headers instrumented with [ASan]. 

```

Comment:
We conventionally use [collapsed reference links](https://github.github.com/gfm/#collapsed-reference-link) instead of [shortcut reference links](https://github.github.com/gfm/#shortcut-reference-link) - they're a bit more verbose but stand out more.